Address referee comments (#145)
gammapy/gammapy-v1.0-paper: src/ms-review.tex

%% CN: I think Fermi-LAT data were available from the 2nd year on
%% https://fermi.gsfc.nasa.gov/ssc/data/policy/
While \fermi data and analysis tools have been public from the

> beginning \citep{}, ground-based \gammaray astronomy has been historically conducted

Reviewer: citation to Atwood 2009 ?

Author: Fixed.

  migration, $\mu=\frac{E}{E_{\rm true}}$.

> \end{itemize}

Reviewer 2: wasn't there a part here, where we said that those counts are put into a Poissonian likelihood that is then minimised to obtian the $\hat{\theta}}$ parameters? Otherwise it's not clear what do we do with the predicted counts. I remember writing this small paragraph.

> DL3 \gammaray data therefore consist of lists of \gammaray-like events and their

Reviewer 2: ```suggestion
The DL3 represent the high-level data from which the analysis previously described starts. This data level therefore contains lists of \gammaray-like events and their
```

Reviewer 2: Then it would transition smoothly in the following subsections, where we say that the Gammapy workflow starts from DL3 data.

Author: The whole section is restructured in https://github.com/adonath/gammapy-v1.0-paper/pull/1. I think this basically includes your comment. But I'll ask one last time for feedback once I have merged the PRs.

> DL3 \gammaray data at therefore consist of lists of \gammaray-like events and their

Reviewer: ```suggestion
DL3 \gammaray data therefore consist of lists of \gammaray-like events and their
```

sub-package which provides a high-level interface for executing analyses
defined from configuration files. In the following sections, we will
introduce all sub-packages and their functionalities in more detail.
In the online documentation we also provide an overview of all

> the \gammapy sub-packages\footnote{\url{https://docs.gammapy.org/1.0/user-guide/package.html}}.q

Reviewer: ```suggestion
the \gammapy sub-packages\footnote{\url{https://docs.gammapy.org/1.0/user-guide/package.html}}.
```
